modules/ossm-support-for-istioctl.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-istioctl-tool
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-support-for-istioctl_{context}"]
+= Support for Istioctl
+
+{SMProduct} 3 provides support for a selection of Istioctl commands.
+
+.Supported Istioctl commands
+[cols="1,1"]
+|===
+|Command |Description 
+
+|`admin` | Manage control plane (`istiod`) configuration 
+|`analyze` | Analyze Istio configuration and print validation messages 
+|`completion` | Generate the autocompletion script for the specified shell
+|`create-remote-secret` | Create a secret with credentials to allow Istio to access remote Kubernetes `apiservers` 
```

Comment:
```suggestion
|`create-remote-secret` | Create a secret with credentials to allow Istio to access remote Kubernetes API servers
```
or keep `apiservers` but follow it with whatever the relevant noun is

Reply:
done